clients/admin-ui/src/features/system/form.ts

```diff
@@ -48,8 +48,7 @@ export const transformSystemToFormValues = (
 ): FormValues => {
   // @ts-ignore
   const dataStewards = system?.data_stewards
-    ?.map((user: any) => user.username)
-    .join(", ");
+    ?.map((user: any) => user.username) || [];
```

Comment:
I know you didn't do this originally, but can we just get rid of the `any` here and use the correct `UserResponse` type (imported from ~/types/api)?
```suggestion
    ?.map((user: UserResponse) => user.username) || [];
```